prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+var declareFirstElement = 'The first element of the array is';
+var firstElement = 'html';
```

Comment:
Here we want to use the array to get the value from a specific `index` of the array. Array are indexed from `0`. So to get the first item in an array we would do:

```JAVASCRIPT
var firstElement = languages[0];
```

Reply:
edited, looks good?

<img width="1440" alt="Screen Shot 2021-09-22 at 4 20 30 PM" src="https://user-images.githubusercontent.com/88061673/134416006-ce552d0c-6880-4f96-b2e5-601460bcf30b.png">